pkg/assets/embed.yml

```diff
@@ -10,8 +10,8 @@ compression:
 
 custom:
   - files:
-    - "../../assets/"
-    base: "../../assets/"
+    - "../../assets/phoenix/"
```

Comment:
Please strip the phoenix folder within the tar command within the makefile. Some consistent release building would be helpful.

Reply:
We are working on the packaging and release scripting for the next phoenix release - yes.

Reply:
Than revert it here and add the strip prefix to tar command :)